### ISSUE TITLE
Cannot Do Anything With Ore Chunks

### ISSUE THREAD

**Issue by ExperimentSixTwoSix** [BloodWorkXGaming/ExNihiloCreatio#136]
I get ore pieces which makes ore chunks but then I cannot craft/smelt or do anything with ore chunks. Is this a bug? Conflicting mods? Am I just stupid and missing some simple step? I cannot find any recipe in JEI using Ore Chunks. I've scoured the internet, including every issue, open and closed here. No one seems to have this issue. I see this mod removed several unnecessary steps from the original Ex Nihilo so I feel like I'm just being stupid and not doing something right. 

All my mods:
Actually Additions
BuildCraft
CodeChicken Lib
CoFH Core
CoFH World
Ex Compressum
Ex Nihilo Creatio
Extra Utilities 2
Forge Mod Loader
Galacticraft Core
Galacticraft Planets
Inventory Tweaks
JourneyMap
Just Enough Items
Mantle
Mekanism
Micdoodle8 Core
Not Enough Items
Redstone Flux
Refined Storage
Thermal Dynamics
Thermal Expansion
Thermal Foundation
Tinkers' Construct
Tiny Progressions

**Comment by SirLyle** [comment]
Is everything running with default configs? I don't see anything off hand that I am aware of which would mess with the smelting recipes. Are you able to smelt other ores?

**Comment by ExperimentSixTwoSix** [comment]
> Is everything running with default configs? I don't see anything off hand that I am aware of which would mess with the smelting recipes. Are you able to smelt other ores?

Yes, I haven't changed anything in the config files. I can smelt the vanilla ores and the ores from other mods without any iss-- Okay, I'm fairly certain I wasn't able to smelt any chunks when I posted this, but now I'm able to smelt all of the chunks except for Osmium. I thought I had to craft the chunks into gravel ore first, but I know I tried smelting chunks/pieces when I realized there was no gravel ore to craft. 

Okay, I can smelt Osmium.. Just not in the Tinkers' Smeltery. You know what, I apologize for wasting your time. I'll just close out this issue and mark it up as me being ignorant.
